[Issue]: App crash on Meta Quest 3 (MQ3)

### Describe the bug

When starting up the moonlight app on the MQ3 OS v76, an app window appears with the message "Moonlight has stopped" and the app will eventually close. Repeat attempts typically do not show any activity other than a white app-launch indicator pip under the recent app shortcuts bar, that shortly fades.

### Steps to reproduce

Assuming an Oculus/Meta Quest headset is available
1) Create a Meta Quest Developer account and enable developer mode on the Meta Quest to allow app sideloading
https://developers.meta.com/horizon/documentation/native/android/ts-odh-getting-started/

2) Join Meta's PTC and update the headset to v76
https://www.meta.com/help/quest/422880102681019/

3) Sideload/Deploy the current .apk build on the headset
https://developers.meta.com/horizon/documentation/unity/ts-odh-deploy-build/

4) Launch the app under the Unknown Sources app group of the Application library in the headset, the error message "Moonlight has stopped" should appear when attempting to first launch the app within the Meta Quest 3 interface.

### Affected games

Not game specific, Moonlight application is unable to start

### Other Moonlight clients

PC

### Moonlight adjusted settings

No

### Moonlight adjusted settings (please complete the following information)

starting from defaults after app sideload

### Moonlight default settings

Yes

### Gamepad-related connection issue

No

### Gamepad-related input issue

No

### Gamepad-related streaming issue

No

### Android version

14

### Device model

Meta Quest 3

### Server PC OS version

Windows 11 24H2

### Server PC GeForce Experience version

11.0.2.341

### Server PC Nvidia GPU driver version

572.83

### Server PC antivirus and firewall software

Windows Defender and Windows Firewall

### Screenshots

_No response_

### Relevant log output

```Shell
03-23 17:44:48.824 10099 10099 E AndroidRuntime: FATAL EXCEPTION: main
03-23 17:44:48.824 10099 10099 E AndroidRuntime: Process: com.limelight, PID: 10099
03-23 17:44:48.824 10099 10099 E AndroidRuntime: java.lang.RuntimeException: Unable to start activity ComponentInfo{com.limelight/com.limelight.PcView}: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.app.GameManager.setGameState(android.app.GameState)' on a null object reference
03-23 17:44:48.824 10099 10099 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3803)
03-23 17:44:48.824 10099 10099 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3943)
03-23 17:44:48.824 10099 10099 E AndroidRuntime: 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
03-23 17:44:48.824 10099 10099 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:139)
03-23 17:44:48.824 10099 10099 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:96)
03-23 17:44:48.824 10099 10099 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2464)
03-23 17:44:48.824 10099 10099 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
03-23 17:44:48.824 10099 10099 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:218)
03-23 17:44:48.824 10099 10099 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:310)
03-23 17:44:48.824 10099 10099 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8228)
03-23 17:44:48.824 10099 10099 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
03-23 17:44:48.824 10099 10099 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
03-23 17:44:48.824 10099 10099 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:978)
03-23 17:44:48.824 10099 10099 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.app.GameManager.setGameState(android.app.GameState)' on a null object reference
03-23 17:44:48.824 10099 10099 E AndroidRuntime: 	at com.limelight.utils.UiHelper$$ExternalSyntheticApiModelOutline6.m(SourceFile:0)
03-23 17:44:48.824 10099 10099 E AndroidRuntime: 	at com.limelight.utils.UiHelper.setGameModeStatus(Unknown Source:41)
03-23 17:44:48.824 10099 10099 E AndroidRuntime: 	at com.limelight.utils.UiHelper.notifyNewRootView(Unknown Source:16)
03-23 17:44:48.824 10099 10099 E AndroidRuntime: 	at com.limelight.PcView.initializeViews(Unknown Source:5)
03-23 17:44:48.824 10099 10099 E AndroidRuntime: 	at com.limelight.PcView.completeOnCreate(Unknown Source:36)
03-23 17:44:48.824 10099 10099 E AndroidRuntime: 	at com.limelight.PcView.onCreate(Unknown Source:51)
03-23 17:44:48.824 10099 10099 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:8636)
03-23 17:44:48.824 10099 10099 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:8614)
03-23 17:44:48.824 10099 10099 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1456)
03-23 17:44:48.824 10099 10099 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3785)
03-23 17:44:48.824 10099 10099 E AndroidRuntime: 	... 12 more
```

### Additional context

Meta Quest OS/Horizon OS v76 on PTC is Android version 14, SDK 34. Not having this issue on my phone which returns the same Android version/SDK using:

adb shell getprop ro.build.version.release
adb shell getprop ro.build.version.sdk

--- a/app/src/main/java/com/limelight/utils/UiHelper.java
+++ b/app/src/main/java/com/limelight/utils/UiHelper.java
@@ -31,13 +31,19 @@ public class UiHelper {
 
     private static void setGameModeStatus(Context context, boolean streaming, boolean interruptible) {
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
-            GameManager gameManager = context.getSystemService(GameManager.class);
+            try {
+                GameManager gameManager = context.getSystemService(GameManager.class);
+                if (gameManager == null) {
+                    return; // Not supported on this device (e.g., Meta Quest)
+                }
 
-            if (streaming) {
-                gameManager.setGameState(new GameState(false, interruptible ? GameState.MODE_GAMEPLAY_INTERRUPTIBLE : GameState.MODE_GAMEPLAY_UNINTERRUPTIBLE));
-            }
-            else {
-                gameManager.setGameState(new GameState(false, GameState.MODE_NONE));
+                if (streaming) {
+                    gameManager.setGameState(new GameState(false, interruptible ? GameState.MODE_GAMEPLAY_INTERRUPTIBLE : GameState.MODE_GAMEPLAY_UNINTERRUPTIBLE));
+                } else {
+                    gameManager.setGameState(new GameState(false, GameState.MODE_NONE));
+                }
+            } catch (Throwable t) {
+                // Swallow any failure. Some OEM builds ship partial/incompatible GameManager impls.
             }
         }
     }

